libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -168,6 +194,14 @@ export class ModelOverview extends React.Component<
       );
     }
 
+    if (this.state.datasetCohortLabeledStatistics.length === 0) {
+      return (
+        <MissingParametersPlaceholder>
+          {"Empty dataset cohort stats"}
```

Comment:
needs to be localized

Also, this isn't really helpful to users I think. How would we arrive in such a situation? There should always be dataset cohorts (at least "All data") so this is really some kind of metric calculation issue then?

Reply:
this was not needed. I removed it.